app/models/event.rb

```diff
@@ -16,6 +16,7 @@ class Event < ActiveRecord::Base
   to_param :title
 
   validates :starts_at, presence: true
+  validates :location_id, presence: true
```

Comment:
@cawel Can we get the validation on `:location` rather than the id, please?

Ernie Miller makes a good case for this in his talk "An Intervention for ActiveRecord by Ernie Miller" -- see the segment "Deep Thoughts on Validations" 

https://youtu.be/yuh9COzp5vo?t=1067